lib/data_update_scripts/20210218041143_backfill_usernames.rb

```diff
@@ -0,0 +1,9 @@
+module DataUpdateScripts
+  class BackfillUsernames
+    def run
+      User.where(username: nil).find_each do |user|
+        user.update(username: "user#{user.id}")
```

Comment:
The username schema is not particularly creative, I just wanted something that's fast and unique. If you'd prefer something else I'm open for all ideas.